tools/docker/docker-compose.yaml

```diff
@@ -32,8 +30,6 @@ services:
       dockerfile: core/chainlink.Dockerfile
     entrypoint: /bin/sh -c "chainlink -c /run/secrets/config node start -d -p /run/secrets/node_password -a /run/secrets/apicredentials"
     restart: always
-    environment:
```

Comment:
Sorry, could you confirm that there is no application that runs "docker-compose -f docker-compose.yaml ..." which assumes that CL_DATABASE_URL is propagated somehow from the upstream?

Reply:
you mean if used without the compose script and hence without docker-compose.postgres.yaml ?

Reply:
yes

Reply:
Its not meant to be used without compose script